endpoints/calendar/index.js

```diff
@@ -43,7 +43,7 @@ const lookupHolidays = (yearStr, monthStr, dayStr) => new Promise((resolve, reje
       return current.date.toISOString().startsWith(`${year}-${month}-${day}`)
     })
     const results = holiday || {
-      date: `${year}-${month}-${day}T00:00:00.000Z`,
+      date: new Date(`${year}-${month}-${day}T00:00:00.000Z`),
```

Comment:
Should we try to use `moment.js` for this so we don't run into some classic JS date problem in the future?